db/migrate/20150106182416_add_votes.rb

```diff
@@ -0,0 +1,10 @@
+class AddVotes < ActiveRecord::Migration
+  def change
+    create_table :votes do |t|
+      t.integer :user_id, null: false
+      t.integer :review_id, null: false
+      t.integer :vote_value, null: false
+    end
+      add_index :votes, :review_id
```

Comment:
You can add an index to ensure that a user can only vote once on a review:

```ruby
add_index :votes, [:review_id, :user_id], unique: true
```

Whenever you have a uniqueness constraint on the model, ensure that it has an accompanying unique index on the database.